README.md

```diff
@@ -5,8 +5,32 @@
 
 **Status**: This project is in active development. We expect to be ready for beta releases in Q2 2025.
 
-The CMIP Rapid Evaluation Framework is a Python application that provides the ability to rapidly process and
-evaluate CMIP data against a set of reference data.
+The [CMIP Rapid Evaluation Framework](https://github.com/CMIP-REF/cmip-ref) is a Python application
+that provides the ability to rapidly process and evaluate CMIP data against a set of reference data.
 This repository contains a set of sample data that can be used to test the REF.
 
 CMIP REF is a community project, and we welcome contributions from anyone.
+
+## Getting started
+
+The sample data is stored in the `data` directory.
+The data is stored in a directory structure that mirrors the structure of the data on the ESGF,
+but each file contains only a subset of the data.
+The `registry.txt` file contains a list of the datasets that are included in the sample data and their hashes.
+These data are fetched by the REF's test suite to ensure a consistent collection of input datasets are used for testing.
+
+The data that are tracked and the decimation process is described in: [scripts/fetch_test_data.py]()
+
+
+### Regenerating the sample data
+
+The file hashes of the generated sample data may vary by platform and various other dependencies.
```

Comment:
I suspect it is not the hashes that are different, but the binary content of the files. If the hashes would be different, it seems likely that pooch would error when trying to download the files.